modules/modules.nix

```diff
@@ -95,6 +96,7 @@ let
     (loadModule ./services/dunst.nix { })
     (loadModule ./services/emacs.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/flameshot.nix { })
+    (loadModule ./services/getmail.nix { })
```

Comment:
Since the service part uses systemd it is limited to Linux, so can add `condition = hostPlatform.isLinux;` here.

Reply:
Done.